models/widgets.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require('mongoose')
+const { Schema } = mongoose
+
+const WidgetSchema = new Schema ({
+  type: String,
+  title: String,
+  size:  {type: Number, required: true},
```

Comment:
Needs spaces between braces and key/value pairs, and I think there's too many spaces between `size` and the opening brace.